docs/basic-docs/atomicdex/atomicdex-tutorials/query-the-mm2-database.md

```diff
@@ -0,0 +1,130 @@
+The AtomicDEX-API stores historical information such as swaps and orders within an SQLite database, located under the DB user data folder (e.g. {folder continaing mm2 binary}/DB/{wallet identifying hex string}/MM2.db).
```

Comment:
@smk762 do you know of a RPC or other way to get the `{wallet identifying hex string}` while mm2 is running?

Reply:
I dont think it's in any RPC - it looks like it's the pubkeyhash. 
I have an online form / API to convert from an address at http://stats.kmd.io/tools/scripthash_from_address/?address=RDragoNHdwovvsDLSLMiAEzEArAD3kq6FN (need to remove the opcodes 76a914 at start and 88ac at end tho).

RPC method would be better - pkh would already be there for communicating with electrum (https://github.com/KomodoPlatform/atomicDEX-API/issues/720)

Added issue to request it's inclusion - https://github.com/KomodoPlatform/atomicDEX-API/issues/1189